mysql-test/suite/galera/t/galera_restart_replica.test

```diff
@@ -1,102 +1,146 @@
 #
-# Test Galera as a replica to a MySQL async replication
+# Test Galera as a replica to a MariaDB async replication
+#
+#   MariaDB
+#    primary ---async replication--->galera node_2 (replica)<----galera replication---> galera node1
+#
+#  Test outline:
+#
+#  - Create user for async replication
+#  - Create table and some data in primary
+#  - Verify that table and data is replicated to galera nodes
+#  - Verify that mysql.gtid_slave_pos has some rows in all Galera nodes
+#  - Verify that gtid_slave_pos, gtid_binlog_pos and gtid_current_pos are
+#    same in all Galera nodes
+#  - Verify that we can shutdown and restart Galera replica (node #2)
```

Comment:
In our previous email exchange, you noted an extended use case for this, where (adapting it to this test) `node_2` would go down, and there would be an SST from `node1`->`node_2` , and then `node_2` would need to restart at the correct slave position. I think it would be good to extend this (or one of your other) test cases to handle that.

Reply:
See line 59, we intentionally force SST (here it does not matter which Galera node is donor) and at line 70 node_2 is restarted and because it has skip-slave-restart=0 slave thread is started.